lib/text/text_engine.js

```diff
@@ -141,6 +142,7 @@ shaka.text.TextEngine = class {
     goog.asserts.assert(
         factory, 'Text type negotiation should have happened already');
     this.parser_ = shaka.util.Functional.callFactory(factory);
+    this.parser_.setSequenceMode(sequenceMode);
```

Comment:
For backward compatibility with external text parsers that predate setSequenceMode, please add an "if" to this to make sure the method exists.  You can `shaka.log.alwaysWarn` if it doesn't.